packages/design-system/src/components/swatch/stories/index.js

```diff
@@ -52,6 +51,14 @@ const Cell = styled.div`
 export default {
   title: 'DesignSystem/Components/Swatch',
   component: Swatch,
+  args: {
+    isDisabled: false,
+  },
+  parameters: {
+    controls: {
+      include: ['isDisabled'],
+    },
+  },
```

Comment:
💭 I think this can be removed. Story already has `disabled` column:

![image](https://user-images.githubusercontent.com/22185279/151244974-83a8181b-eb51-4b0e-a8b9-d15d7428b732.png)